tests/php/ImageTest.php

```diff
@@ -532,6 +532,36 @@ public function testGetSetImageBackend()
         $this->assertEquals(99999, $image->getWidth());
     }
 
+    public function testLazyLoad()
+    {
+        Config::modify()->set(Image::class, 'lazy_loading_enabled', true);
+
+        /** @var Image $image */
+        $image = $this->objFromFixture(Image::class, 'imageWithTitle');
+        $this->assertTrue($image->getIsLazyLoaded(), 'Images lazy load by default');
+
+        $expected = '<img src="/assets/ImageTest/folder/test-image.png" alt="This is a image Title" loading="lazy" width="300" height="300" />';
+        $actual = trim($image->getTag());
+        $this->assertEquals($expected, $actual, 'Lazy load img tag renders correctly');
+
+        $image->setLazyLoad(false);
+        $this->assertFalse($image->getIsLazyLoaded(), 'Images can be eager loaded on a per-image basis');
+
+        $image->setLazyLoad(true);
+        Config::modify()->set(Image::class, 'lazy_loading_enabled', false);
```

Comment:
You could [use the new withConfig API](https://github.com/silverstripe/silverstripe-framework/commit/e2bea6b41f05f2ad9d492d2e96a535dfff9f16ef) here